public/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="css/main.css">
+	<!-- Load the Paper.js library -->
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
+	<script type="text/javascript" src="/js/paper.js"></script>
+	<script src="https://cdn.socket.io/socket.io-1.2.0.js"></script>
```

Comment:
There's probably a better way to do this besides using a CDN for sockets. The plain socket.io is supposed to give access to socket.io-client already, and should find out why the other way isn't working first, especially since such a large part of this will rely on sockets.